packages/blockchain-wallet-v4-frontend/src/modals/SimpleBuy/EnterAmount/Checkout/validation.tsx

```diff
@@ -1,37 +1,32 @@
 import { convertBaseToStandard } from 'data/components/exchange/services'
-import { SBCheckoutFormValuesType } from 'data/types'
+import { SBCheckoutFormValuesType, SBFormPaymentMethod } from 'data/types'
 import BigNumber from 'bignumber.js'
 
 import { SBPairType } from 'core/types'
 
 export const getMaxMin = (
   pair: SBPairType,
   minOrMax?: 'min' | 'max',
-  allValues?: SBCheckoutFormValuesType
+  allValues?: SBCheckoutFormValuesType,
+  method?: SBFormPaymentMethod
 ) => {
   switch (minOrMax || 'max') {
     case 'max':
       const defaultMax = convertBaseToStandard('FIAT', 0)
       if (!allValues) return defaultMax
-      if (!allValues.method) return defaultMax
+      if (!method) return defaultMax
       if (!pair) return defaultMax
 
-      const max = BigNumber.minimum(
-        allValues.method.limits.max,
-        pair.buyMax
-      ).toString()
+      const max = BigNumber.minimum(method.limits.max, pair.buyMax).toString()
 
       return convertBaseToStandard('FIAT', max)
     case 'min':
       const defaultMin = convertBaseToStandard('FIAT', 0)
       if (!allValues) return defaultMin
-      if (!allValues.method) return defaultMin
+      if (!method) return defaultMin
       if (!pair) return defaultMin
 
-      const min = BigNumber.maximum(
-        allValues.method.limits.min,
-        pair.buyMin
-      ).toString()
+      const min = BigNumber.maximum(method.limits.min, pair.buyMin).toString()
```

Comment:
This may lead to a weird UX because in the past we always had a method, now we only have a method if a user previously made an order. Let's keep an eye on this.

Reply:
We decided that if no method is selected there should not be a check here.